android/AndroidManifest.xml

```diff
@@ -4,6 +4,8 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:installLocation="auto">
 
+
```

Comment:
Please remove/discard any whitespace changes. Which Git editor do you use? Git Gui?